src/mn_setup.c

```diff
@@ -1876,6 +1877,10 @@ static void RefreshSolidBackground(void)
     st_refresh_background = true;
 }
 
+static const char *hud_anchoring_strings[] = {
+    "Wide", "4:3", "16:9", "21:9"
```

Comment:
I'll probably remove 21:9 since it would rarely be selected (users who have a 32:9 monitor and want 21:9 HUD anchoring for some reason). This was also suggested [here](https://github.com/fabiangreffrath/woof/commit/4b523c00e6b5b6f40b22b2eb033eacb261247caa#commitcomment-164167854).

Reply:
> I think the 21:9 option is okay.

Okay, I'll leave it in.

> Should we select 16:9 by default instead of "Wide'" (auto)?

Sure, fixed here: https://github.com/fabiangreffrath/woof/pull/2511/commits/0bf9ced26739653690497d3195618aa7b8f50b80